### PR TITLE
feat(detector): add known exploited vulnerabilities

### PR DESCRIPTION
**Pull request by MaineK00n**
# What did you implement:

Information from the Known Exploited Vulnerabilities Catalog provided by the Cybersecurity & Infrastructure Security Agency will be made available.

## NOTE
Changed the struct tag(`` `json:""` ``) in this PR.
https://github.com/future-architect/vuls/pull/1331/files#diff-25e484c3ec31cd05d7ce8ab0ab61b6eea8b82ecd47f9cf62e7448e56fb8d6e4dL819-R824

So, if you read ("report", "tui" command) the results.json from the old version of vuls with the new version of vuls, the CERT Alert will be blank.
To display the Alert correctly, use the `-refresh-cve` option when using the report and tui commands.
This is also the case when reading results.json created by newer versions of vuls with older versions.

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
## vuls report
```console
$ vuls report
[Nov 19 09:07:18]  INFO [localhost] vuls-v0.18.1-build-20211119_090247_c0379f3
[Nov 19 09:07:18]  INFO [localhost] Validating config...
[Nov 19 09:07:18]  INFO [localhost] cveDict.type=sqlite3, cveDict.url=, cveDict.SQLite3Path=/usr/share/vuls-data/cve.sqlite3
[Nov 19 09:07:18]  INFO [localhost] ovalDict.type=sqlite3, ovalDict.url=, ovalDict.SQLite3Path=/usr/share/vuls-data/oval.sqlite3
[Nov 19 09:07:18]  INFO [localhost] gost.type=sqlite3, gost.url=, gost.SQLite3Path=/usr/share/vuls-data/gost.sqlite3
[Nov 19 09:07:18]  INFO [localhost] exploit.type=sqlite3, exploit.url=, exploit.SQLite3Path=/usr/share/vuls-data/go-exploitdb.sqlite3
[Nov 19 09:07:18]  INFO [localhost] metasploit.type=sqlite3, metasploit.url=, metasploit.SQLite3Path=/usr/share/vuls-data/go-msfdb.sqlite3
[Nov 19 09:07:18]  INFO [localhost] kevuln.type=sqlite3, kevuln.url=, kevuln.SQLite3Path=/usr/share/vuls-data/go-kev.sqlite3
[Nov 19 09:07:18]  INFO [localhost] Loaded: /home/mainek00n/github/github.com/MaineK00n/vuls/results/2021-11-19T09:06:14+09:00
[Nov 19 09:07:18]  INFO [localhost] No need to refresh
[Nov 19 09:07:18]  INFO [localhost] vuls-target: total 254 CVEs detected
[Nov 19 09:07:18]  INFO [localhost] vuls-target: 29 CVEs filtered by --confidence-over=80
vuls-target (debian10.11)
=========================
Total: 225 (Critical:20 High:79 Medium:95 Low:16 ?:15)
0/222 Fixed, 67 poc, 0 exploits, cisa 2 alerts, cert(en: 4, ja: 6)
218 installed

+---------------------+------+--------+-----+-----------+---------+---------------------------------------------------+
|       CVE-ID        | CVSS | ATTACK | POC |   ALERT   |  FIXED  |                        NVD                        |
+---------------------+------+--------+-----+-----------+---------+---------------------------------------------------+
| CVE-2016-1585       |  9.8 |  AV:N  |     |           | unfixed | https://nvd.nist.gov/vuln/detail/CVE-2016-1585    |
...
| CVE-2021-42013      |  9.8 |  AV:N  | POC | CISA/CERT |         | https://nvd.nist.gov/vuln/detail/CVE-2021-42013   |
...
| CVE-2021-41524      |  7.5 |  AV:N  |     |      CERT |         | https://nvd.nist.gov/vuln/detail/CVE-2021-41524   |
| CVE-2021-41773      |  7.5 |  AV:N  | POC | CISA/CERT |         | https://nvd.nist.gov/vuln/detail/CVE-2021-41773   |
| CVE-2008-4609       |  7.1 |  AV:N  |     |      CERT | unfixed | https://nvd.nist.gov/vuln/detail/CVE-2008-4609    |
...
```

## vuls tui
Screenshot from 2021-11-19 09-10-50

# Checklist:
You don't have to satisfy all of the following.

- [ ] Write tests
- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES

# Reference
* https://github.com/vulsio/go-kev
* https://github.com/vulsio/vulsctl/pull/50
* https://github.com/vulsdoc/vuls/pull/176

### CODE REVIEW

File: models/vulninfos.go [hunk 1]
```diff
@@ -807,25 +806,39 @@ type Metasploit struct {
...
 	URL            string         `json:"url,omitempty"`
 }
 
-// AlertDict has target cve JPCERT and USCERT alert data
+// AlertDict has target cve JPCERT, USCERT and CISA alert data
 type AlertDict struct {
-	Ja []Alert `json:"ja"`
-	En []Alert `json:"en"`
+	Ja   []Alert `json:"ja"`
+	En   []Alert `json:"en"`
```
Comment (MaineK00n) [inline models/vulninfos.go:825]: I want to use `json: "cert(jp)"`, `json: "cert(us)"`.

Comment (MaineK00n) [reply models/vulninfos.go:825]: Will it affect backward compatibility?

File: models/vulninfos.go [hunk 2]
```diff
@@ -814,18 +813,30 @@ type Mitigation struct {
...
+		s = "CISA"
 	}
-	return ""
+	if len(a.USCERT) != 0 || len(a.JPCERT) != 0 {
+		if s != "" {
+			s += "/"
+		}
+		s += "CERT"
+	}
+	return s
```
Comment (sadayuki-matsuno) [inline models/vulninfos.go:839]: 私ならこう書きます！
```
var s []string
if len(a.CISA) != 0 {
	s = append(s, "CISA")
}
if len(a.USCERT) != 0 || len(a.JPCERT) != 0 {
	s = append(s, "CERT")
}
return s.Join(s, "/")
```

Comment (MaineK00n) [reply models/vulninfos.go]: Review has been reflected.

**Pull request merged**